internal/app/parcel_carrier/service.go

```diff
@@ -0,0 +1,24 @@
+package parcel_carrier
```

Comment:
package name `carrier` is good enough to understand as there is no other carrier in this service